src/include/connection.h

```diff
@@ -287,6 +287,21 @@ typedef TAILQ_HEAD(__wt_backuphash, __wt_backup_target) WT_BACKUPHASH;
 
 extern const WT_NAME_FLAG __wt_stress_types[];
 
+#define WT_SESSION_FOREACH_BEGIN(session, conn)                                                \
```

Comment:
Possibly rename to `WT_CONN_SESSION_FOREACH_BEGIN`, or relocate this to `session.h`.